subprojects/platform-jvm/src/main/java/org/gradle/api/tasks/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Base JVM task-related API.
+ */
+package org.gradle.api.tasks;
```

Comment:
Do we really need to re-define this `package-info` if it already exists in another project? The build complains otherwise.

Reply:
It is not needed. There is a file which is used for disabling this check for split packages